package/src/kustoMode.ts

```diff
@@ -43,8 +57,14 @@ export function setupMode(defaults: LanguageServiceDefaultsImpl, monacoInstance)
             (worker) =>
                 ({
                     ...worker,
-                    setSchema: (schema) => augmentedSetSchema(schema, worker),
+                    setSchema: (schema) => {
+                        getKustoWorkerTokenParsing().then(worker => worker(first, ...more).then(wor => augmentedSetSchema(schema, wor)));
```

Comment:
I assume setting up additional worker come with performance penalty (double loading/updates)
consider adding parameter to setting or setupMode ( i would keep the default as disabled)

Reply:
will do